acquisition/tools/acq_checkTriggers.m

```diff
@@ -7,24 +7,25 @@ function acq_checkTriggers(bbci)
 %If no BBCI structure is specified, the default bbci.source is assumed
 %which is (currently) BrainVision.
 
-
-misc_checkTypeifExists('bbci', 'STRUCT');
+% misc_checkTypeIfExists('bbci', 'STRUCT');
+% The above function will never throw an error!
```

Comment:
That is strange. It should not throw an error. Then we have to fix the function misc_checkTypeIfExists.